src/validation/loader.rb

```diff
@@ -35,11 +35,13 @@ def initialize
         @path = FilePath
       end
 
-      def configure_data
+      # TODO: Remove references to configure_data
+      def question_tree
         # XXX Extract object for loading configure data?
-        @configure_data ||=
+        @questions ||=
           Validation::Configure.new(combined_configure_data).tree
       end
+      alias configure_data question_tree
```

Comment:
Since Metalware itself is the only dependency on this class, I don't think we need to alias this method so we now have 2 names for it. If you want to rename an internal method I think it would be better to just rename it, rather than half rename it while still having the old name work. Unneccesarily having 2 names for the same thing only makes the codebase slightly more complicated and harder to follow. There's not going to be an easier time to rename it entirely than at the point when you come up with a new name for it - in future there's only likely to be more references to it and we're likely to be less familiar with the code and the original reason there's 2 names for the same method.

The only exception to this I can think of is if the method is public to the project and there are external dependencies on it, in which case you should be a bit more cautious and gradually deprecate it if the rename is actually important, to avoid breaking other code which you do not control and cannot fix.